source/treeInterceptorHandler.py

```diff
@@ -144,22 +144,27 @@ def makeTextInfo(self,position):
 
 class RootProxyTextInfo(textInfos.TextInfo):
 
-	def __init__(self,obj,position,_rangeObj=None):
+	def __init__(self,obj,position,**kwargs):
 		super(RootProxyTextInfo,self).__init__(obj,position)
-		self.innerTextInfo=self.InnerTextInfoClass(obj.rootNVDAObject,position,_rangeObj=_rangeObj)
-
-	def _get_InnerTextInfoClass(self):
-		return self.obj.rootNVDAObject.TextInfo
+		self.InnerTextInfoClass=self.obj.rootNVDAObject.TextInfo
+		if isinstance(position,self.InnerTextInfoClass):
+			self.innerTextInfo=position
+		else:
+			self.innerTextInfo=self.InnerTextInfoClass(obj.rootNVDAObject,position,**kwargs)
 
 	def copy(self):
-		return self.__class__(self.obj,None,_rangeObj=self.innerTextInfo._rangeObj)
+		innerCopy=self.innerTextInfo.copy()
+		return self.__class__(self.obj,innerCopy)
 
 	def _get__rangeObj(self):
 		return self.innerTextInfo._rangeObj
 
 	def _set__rangeObj(self,r):
 		self.innerTextInfo._rangeObj=r

```

Comment:
I think so, yes. There may be still code in UIA or MS word browse modes that may depend on accessing it. We should look into removing them if logically possible, but not in this particular PR. The change here was to specifically allow copying with no _rangeObj.